doc/uploading-files.md

```diff
@@ -0,0 +1,55 @@
+## Uploading files
+
+Lets say you want your blog posts to contain images.
+Here's how you add file-upload in `data/collection/post.json`:
+
+    "properties":{
+
+      ...
+
+      "picture": {
+          "title": "Picture",
+          "type": "string",
+          "media": {
+              "binaryEncoding": "base64",
+              "type": "image/png"
+          },
+          "links": [
+             {
+               "href": "/custom_endpoint/{{self}}.png",
+               "mediaType": "image/png"
+             }
+          ]
+      }
+
+Or generic files:
+
+    "properties":{
+
+      ...
+
+      "file": {
+        "type": "string",
+        "format": "file",
+        "title": "File"
+        "links": [
+          {
+            "rel": "Download File",
+            "href": "/custom_endpoint/{{self}}",
+            // Can also set `download` to a string as per the HTML5 spec
+            "download": true
+          }
+        ]
+      }
+
+> NOTE: the links probably will need a custom endpoint which serves the file with the proper media type 
+
+    expressa.get('/files/:file',function(req,res,next){
```

Comment:
Yes, well I found out file-uploads are already handled in json-editor (is there anything this library CAN'T do ? lol).
Indeed it basically converts files to base64.
I kindof like the KiSS aspect of it.
And yes, for the project im working on right now, i need file-upload.
So I'll probably update the docs later as my project progresses.
I can also imagine an extra example of a expressa listener which:

* decodes the base64 from the file-property, and stores it into a folder (or S3 bucket e.g.)
* empties the file-property

The 'links'-property (see json-editor docs) can then just create a preview- or link of the particular file.
Tons of possibilities here, I think we can really benefit from json-editor's file-upload instead of rolling our own.